Policies/github-terms-of-service.md

```diff
@@ -46,6 +46,7 @@ Effective date:
 4. “The User,” “You,” and “Your” refer to the individual person, company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the account; or that directs the use of the account in the performance of its functions. A User must be at least 13 years of age. Special terms may apply for business or government accounts (See [Section B(4): Additional Terms](#4-additional-terms)).
 5. “GitHub,” “We,” and “Us” refer to GitHub, Inc., as well as our affiliates, directors, subsidiaries, contractors, licensors, officers, agents, and employees.
 6. “Content” refers to content featured or displayed through the Website, including without limitation text, data, articles, images, photographs, graphics, software, applications, designs, features, and other materials that are available on the Website or otherwise available through the Service. "Content" also includes Services. “User-Generated Content” is Content, written or otherwise, created or uploaded by our Users. "Your Content" is Content that you create or own. “Paid Content” is Content only available to Users who are participating in a payment plan, including private repositories.
+7. “Free Licence” refers to a Free / Open Source / Open Knowledge / Ⓕ Copyfree licence. For practicality reasons, the set of Free Licences is comprised of the licences listed at [ifrOSS](http://www.ifross.org/en/license-center), [Debian](https://wiki.debian.org/DFSGLicenses), [OKFN](http://opendefinition.org/licenses/), [OSI](https://opensource.org/licenses/alphabetical), [FSF](https://www.gnu.org/licenses/license-list.html), [the Copyfree Initiative](http://copyfree.org/standard/licenses), and/or the [list of licences approved for Free Cultural Works](http://freedomdefined.org/Licenses), explicitly excluding those listed on these sites as not Free (such as those under the headings “B. Other Software Licenses”, “D. Other Licenses for Fair Use of Immaterial Goods”, “DFSG-incompatible”, “Non-Conformant Licenses”, “Nonfree Software Licenses”, “Nonfree Documentation Licenses”, “Licenses for Works stating a Viewpoint”, “Invariant sections” in the linked lists).
```

Comment:
Is there an existing term for this, or is “Free License” something we're making up for this ToS?  What happens to GitHub and its users if one of the organizations you cite changes its criteria?  Also note that the pages you link for approved licenses may not be complete.  For example, the [OSI has approved the AFL-1.0 (since superseded by the AFL-3.0)][1], but the AFL-1.0 currently shows up on neither of the OSI's [license-list][2] [pages][3].

I don't see a way around that short of defining what GitHub needs and what other users need (e.g. “[free redistribution][4]” for forking public repositories and “[derived works][5]” for creating public branches to be used in pull requests) or explicitly whitelisting licenses which GitHub considers sufficient.

[1]: https://lists.opensource.org/pipermail/license-discuss/2017-July/019922.html
[2]: https://opensource.org/licenses/alphabetical
[3]: https://opensource.org/licenses/do-not-use.html
[4]: https://opensource.org/osd#free-redistribution
[5]: https://opensource.org/osd#derived-works